src/telliot_feeds/sources/manual_sources/diva_manual_source.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+from typing import List
+
+from telliot_feeds.datasource import DataSource
+from telliot_feeds.dtypes.datapoint import datetime_now_utc
+from telliot_feeds.dtypes.datapoint import OptionalDataPoint
+from telliot_feeds.utils.log import get_logger
+
+
+logger = get_logger(__name__)
+
+
+def _helper(asset_collateral: str) -> float:
+    asset_collateral_price = None
+    print(f"Type price of the {asset_collateral} and press [ENTER]")
+
+    while asset_collateral_price is None:
+        inpt = input()
+        try:
+            price = float(inpt)
+        except ValueError:
+            print(f"Invalid {asset_collateral} price input. Enter decimal value (float).")
+            continue
+
+        asset_collateral_price = price
+    return asset_collateral_price
+
+
+@dataclass
+class DivaManualSource(DataSource[List[float]]):
+    """Datasource for Diva Protocol values"""
+
+    def parse_user_val(self) -> List[float]:
+        vals = ["reference asset", "collateral token"]
+
+        prices = [_helper(i) for i in vals]
+
+        with_precision = [int(i * 1e18) for i in prices]
+        print(
+            "\nDiva protocol reference asset and collateral token prices (with 18 decimals of precision) "
+            + f"to be submitted on chain: {with_precision}"
```

Comment:
Can you include a test to check the formatting of this printed to the user? We want to avoid it printing scientific notation in the list